drivers/enc28j60/enc28j60.c

```diff
@@ -34,6 +34,10 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
+#define SPI_BUS         (dev->p.spi)
```

Comment:
I don't think this is any cleaner. It also makes it much harder to change code using tools, compared to not using a define.

In order to reduce duplication, you could use per-function variables.

Reply:
thats how its mostly done right now for many drivers, which doesn't make it (more) right but at least consistent 😉

Reply:
Well, it is done for about a third of all drivers... I see this is a more general discussion, so ok.